### DISCUSSION TITLE
Building from source on linux - how to enable SDK_TOBII?

### DISCUSSION THREAD

**Discussion by rykugur** [opentrack/opentrack#1832] [Q&A] [answered]
I was informed that tobii support was added to opentrack, and that specifically the Tobii Eyetracker 5 is now working.

I'm trying to build from source on linux (nixOS), but am not sure what I need to do to "include" tobii support. I tried:

```
mkdir build
cd build
cmake ..
ccmake .
```
But the `SDK_TOBII` option is not toggle-able. I tried passing `-DSDK_TOBII=ON` (similar to the way I passed `-DSDK_WINE=ON`), but that did not seem to do anything. `ccmake` screenshot:

![image](https://github.com/opentrack/opentrack/assets/2609898/ce06eaf1-7650-4849-bf76-9ff9925e9218)

What do I need to do to "include" tobii support?

**Comment by sthalik** [comment, marked as answer]
It's a mistake that `SDK_TOBII` gets set on non-win32. The SDK isn't available for non-Windows platforms. See: https://developer.tobii.com/product-integration/stream-engine/getting-started/

**Reply by rykugur** [reply]
Thank you for the quick response.
